lms/djangoapps/grades/management/commands/backfill_grades.py

```diff
@@ -0,0 +1,141 @@
+"""
+Command to compute all grades for specified courses.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+import six
+
+from openedx.core.lib.command_utils import (
+    get_mutually_exclusive_required_option,
+    validate_dependent_option,
+    parse_course_keys,
+)
+from student.models import CourseEnrollment
+from xmodule.modulestore.django import modulestore
+
+from ... import tasks
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        $ ./manage.py lms backfill_grades --all_courses --settings=devstack
```

Comment:
Done.  I had to add a commit (which I'm going to keep separate from the one that introduces this work) to remove the existing broken compute_grades command.